Corona-Warn-App/src/main/res/values-de/event_registration_strings.xml

```diff
@@ -98,4 +98,11 @@
     <!-- XTXT: Title for other event  -->
     <string name="tracelocation_organizer_category_other_event_title">anderes Event</string>
 
+
+    <!--    Event Registration Homescreen Card-->
+    <!-- XHED: Headline for event registration creation card -->
+    <string name="create_event_card_headline">"Sie planen eine\nVeranstaltung?"</string>
```

Comment:
Why `\n` here?

Reply:
Because the design had "Veranstaltung" on a new line no matter the text size. Otherwise, the text would fit on a single row. But I wouldn't focus too much on texts since they aren't final and they're different between iOS and Android.